config/v1/types_dns.go

```diff
@@ -53,6 +53,13 @@ type DNSSpec struct {
 	//
 	// +optional
 	PrivateZone *DNSZone `json:"privateZone,omitempty"`
+	// platform holds configuration specific to the underlying
+	// infrastructure provider for DNS.
+	// When omitted, this means the user has no opinion and the platform is left
+	// to choose reasonable defaults. These defaults are subject to change over time.
+	//
+	// +optional
```

Comment:
For faster merge as we re-open the queue, make this field contingent on TechPreview like https://github.com/openshift/api/blob/master/example/v1/types_stable.go#L36.

You'll need to pre-create the CRD manifest as we did for infrastructure https://github.com/openshift/api/blob/master/config/v1/0000_10_config-operator_01_infrastructure-TechPreviewNoUpgrade.crd.yaml

Reply:
> You'll need to pre-create the CRD manifest as we did for infrastructure https://github.com/openshift/api/blob/master/config/v1/0000_10_config-operator_01_infrastructure-TechPreviewNoUpgrade.crd.yaml

How should I pre-create this? I tried seeding the tech preview manifest by `cp`ing the default CRD and appending `TechPreviewNoUpgrade` to the filename, hoping that `make update` would pick it up, but it didn't seem to work (6e1722fb0a52c0569bca3ac52769db433da2e0df).

Reply:
missing the `release.openshift.io/feature-set: TechPreviewNoUpgrade` annotation.